fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/rdf/impl/PropertiesRdfContext.java

```diff
@@ -66,7 +68,14 @@ public PropertiesRdfContext(final FedoraResource resource,
     private Iterator<Triple> triplesFromProperties(final FedoraResource n)
         throws RepositoryException {
         LOGGER.trace("Creating triples for node: {}", n);
-        final Iterator<Property> allProperties = n.getNode().getProperties();
+        final Iterator<Property> allProperties;
+        if (n instanceof FedoraBinary) {
```

Comment:
In the case when the FedoraResource is a FedoraBinary, you should be able to concatenate the properties from both the description and the arg node itself. The end.